### PR TITLE
Pass Chain icon to embed via query param

### PR DESCRIPTION
**Pull request by MananTank**

### CODE REVIEW

File: contract-ui/tabs/embed/components/embed-setup.tsx [hunk 1]
```diff
@@ -30,6 +30,21 @@ import {
   Heading,
 } from "tw-components";
 
+export function minimizeChain(chain: Chain) {
```
Comment (jnsdls) [inline contract-ui/tabs/embed/components/embed-setup.tsx:33]: let's instead change this in the package please

File: .env.example [hunk 2]
```diff
@@ -6,6 +6,9 @@ NEXT_PUBLIC_THIRDWEB_API_HOST="http://localhost:3005"
 # staging
 NEXT_PUBLIC_THIRDWEB_API_HOST="https://api.staging.thirdweb.com"
 
+# Get the client id from https://thirdweb.com/create-api-key and set it here, Make sure to allow localhost:3000 in the allowed origins
+NEXT_PUBLIC_DASHBOARD_CLIENT_ID=
```
Comment (jnsdls) [inline .env.example:10]: great!

**Review by jnsdls — CHANGES_REQUESTED** [review]
let's move the change to the chains package

File: contract-ui/tabs/embed/components/embed-setup.tsx (hunk 1 above)
Comment (MananTank) [reply contract-ui/tabs/embed/components/embed-setup.tsx:33]: Moved it to dashboard because that function is only used in dashboard

Comment (MananTank) [reply contract-ui/tabs/embed/components/embed-setup.tsx]: Done

**Review by jnsdls — APPROVED** [review]
LGTM

**Pull request merged**